netkit/http/version.nim

```diff
@@ -43,4 +35,8 @@ proc parseHttpVersion*(s: string): HttpVersion  {.raises: [ValueError].} =
     if name[i] != s[i]:
       raise newException(ValueError, "Invalid Http Version")
     i.inc()
-  result = (s, major.Natural, minor.Natural)
+  case minor
```

Comment:
maybe `result = HttpVersion[minor]` is better.

Reply:
added a little optimization